HolyshiftService/JwtService.cs

```diff
@@ -0,0 +1,96 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+public class JwtService
+{
+    private readonly byte[] _keyBytes = new byte[32];
+    private readonly string _issuer;
+    private readonly string _audience;
+    private readonly int _accessTokenExpirationMinutes;
+    private readonly int _refreshTokenExpirationDays;
+    private readonly ILogger<JwtService> _logger;
+
+    public JwtService(IConfiguration configuration, ILogger<JwtService> logger)
+    {
+        var jwtSettings = configuration.GetSection("JwtSettings");
+        var key = jwtSettings["Key"];
+        var issuer = jwtSettings["Issuer"];
+        var audience = jwtSettings["Audience"];
+        var accessTokenExpirationMinutes = int.Parse(jwtSettings["AccessTokenExpirationMinutes"]);
+        var refreshTokenExpirationDays = int.Parse(jwtSettings["RefreshTokenExpirationDays"]);
+
+        _issuer = issuer;
+        _audience = audience;
+        _accessTokenExpirationMinutes = accessTokenExpirationMinutes;
+        _refreshTokenExpirationDays = refreshTokenExpirationDays;
+        _logger = logger;
+
+        using (var rng = RandomNumberGenerator.Create())
+        {
+            rng.GetBytes(_keyBytes);
+        }
+    }
+
+
+    public string GenerateJwtToken(HolyShiftUser user)
+    {
+        _logger.LogInformation("Generating JWT token for user: " + user.Email);
+
+        var securityKey = new SymmetricSecurityKey(_keyBytes);
+        var credentials = new SigningCredentials(securityKey, SecurityAlgorithms.HmacSha256);
```

Comment:
FYI, this is good but there is better approach but harder to implement.

The better approach is to use asymmetric keys (e.g. rsa).

This is fine for now, but keep in mind we might migrate to asymmetric keys in the future.

In click up, add "TechnicalDebt" list and a task to do this migration there.